### PR TITLE
Shirisha addition

### PR DESCRIPTION
**Pull request by Shirisha-CA**
Spelling mistake

### CODE REVIEW

**Comment by Shirisha-CA** [comment]
Typo mistake

File: calculator.html [hunk 1]
```diff
@@ -1 +1,2 @@
-<h2>Hi Evereyone!!!</h2>
\ No newline at end of file
+<h2>Hi Evereyone!!!</h2>
```
Comment (rushendraChowdary) [inline calculator.html:1]: typo

Comment (Shirisha-CA) [reply calculator.html:1]: Ok

File: calculator.html [hunk 2]
```diff
@@ -1 +1,2 @@
...
+<h2>Shirisha Implement the Adtion</h2>
```
Comment (rushendraChowdary) [inline calculator.html]: typo in addition

File: calculator.html (hunk 1 above)
Comment (rushendraChowdary) [inline calculator.html:1]: typo in everyone

**Comment by Shirisha-CA** [comment]
I corrected my mistakes

**Pull request merged**